.github/ISSUE_TEMPLATE/issue_pm_template.md

```diff
@@ -0,0 +1,18 @@
+---
+name: ☁ Governance management 🎂
+about: Indicate an issue with the project management of CAMARA
+title: ''
+labels: 'project management'
+assignees: ''
+
+---

```

Comment:
I suggest that the Marketing working group is not responsible for Governance issues ... here the typical repository management issue:

```suggestion
---
name: ☁ Repository management 🎂
about: Indicate an issue with repository or release management
title: ''
labels: 'repository management'
assignees: ''
---
```